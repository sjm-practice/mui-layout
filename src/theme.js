import { createMuiTheme } from "@material-ui/core/styles";
import { red } from "@material-ui/core/colors";

// Create a theme instance.
const theme = createMuiTheme({
  palette: {
    error: {
      main: red.A400,
    },
  },
});

export default theme;
