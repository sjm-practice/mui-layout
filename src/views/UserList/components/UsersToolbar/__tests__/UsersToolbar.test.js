/* eslint-env jest */

import React from "react";
import { render } from "@testing-library/react";
import { ThemeProvider } from "@material-ui/styles";
import theme from "../../../../../theme";

import UsersToolbar from "../index";

describe("<UsersToolbar />", () => {
  it("Should match Snapshot", () => {
    const { asFragment } = render(
      <ThemeProvider theme={theme}>
        <UsersToolbar />
      </ThemeProvider>,
    );

    expect(asFragment()).toMatchSnapshot();
  });
});
