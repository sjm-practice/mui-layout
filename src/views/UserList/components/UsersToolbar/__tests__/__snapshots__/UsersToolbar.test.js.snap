// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<UsersToolbar /> Should match Snapshot 1`] = `
<DocumentFragment>
  <div
    class="makeStyles-root-1"
  >
    <div
      class="makeStyles-row-2"
    >
      <span
        class="makeStyles-spacer-3"
      />
      <button
        class="MuiButtonBase-root MuiButton-root makeStyles-importButton-4 MuiButton-text"
        tabindex="0"
        type="button"
      >
        <span
          class="MuiButton-label"
        >
          Import
        </span>
        <span
          class="MuiTouchRipple-root"
        />
      </button>
      <button
        class="MuiButtonBase-root MuiButton-root makeStyles-exportButton-5 MuiButton-text"
        tabindex="0"
        type="button"
      >
        <span
          class="MuiButton-label"
        >
          Export
        </span>
        <span
          class="MuiTouchRipple-root"
        />
      </button>
      <button
        class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary"
        tabindex="0"
        type="button"
      >
        <span
          class="MuiButton-label"
        >
          Add user
        </span>
        <span
          class="MuiTouchRipple-root"
        />
      </button>
    </div>
    <div
      class="makeStyles-row-2"
    >
      <div
        class="MuiPaper-root MuiPaper-elevation1 makeStyles-root-27 makeStyles-searchInput-6 MuiPaper-rounded"
        placeholder="Search user"
      >
        <svg
          aria-hidden="true"
          class="MuiSvgIcon-root makeStyles-icon-28"
          focusable="false"
          role="presentation"
          viewBox="0 0 24 24"
        >
          <path
            d="M15.5 14h-.79l-.28-.27C15.41 12.59 16 11.11 16 9.5 16 5.91 13.09 3 9.5 3S3 5.91 3 9.5 5.91 16 9.5 16c1.61 0 3.09-.59 4.23-1.57l.27.28v.79l5 4.99L20.49 19l-4.99-5zm-6 0C7.01 14 5 11.99 5 9.5S7.01 5 9.5 5 14 7.01 14 9.5 11.99 14 9.5 14z"
          />
        </svg>
        <div
          class="MuiInputBase-root MuiInput-root makeStyles-input-29"
        >
          <input
            class="MuiInputBase-input MuiInput-input"
            placeholder="Search user"
            type="text"
            value=""
          />
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
